### PR TITLE
About page added.

### PR DESCRIPTION
**Pull request by medhach**
Added functionality for the About button

### CODE REVIEW

File: PowerUp/app/src/main/java/powerup/systers/com/AboutPageActivity.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import android.os.Bundle;
+import android.widget.TextView;
+
+public class AboutPageActivity extends Activity {
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.about_page);
+        TextView content = (TextView) findViewById(R.id.aboutPageContent);
+        content.setText("" + getString(R.string.about_page_content));
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/AboutPageActivity.java]: A (" " +) concatenation is not required when you are fetching a string itself.

File: PowerUp/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -18,5 +18,8 @@
     <string name="back_to_map">Back to Map</string>
     <string name="replay">Replay</string>
     <string name="preferences_has_previously_started">has_previously_started</string>
+    <string name="about_page_content">This app was built through a Syster\'s initiative, to improve the physical and mental health trends of pre-adoloescents all over the world. PowerUp teaches social, emotional learning, conflict resolution, and health skills in a fun, interactive way. Through the game, the player\'s decision making skills and trends while in critical situations are judged and alloted points.
+    Super powers can be unleashed to make the player stronger as the game proceeds.</string>
+    <string name="about_page_header">About PowerUp</string>
```
Comment (chhavip) [inline PowerUp/app/src/main/res/values/strings.xml]: The content can be broken down into sections and have some more amount of information displayed.

**Comment by anubhakushwaha** [comment]
@chhavip Please close this PR

**Comment by aanchal07** [comment]
Closing the pull request since there are conflicts.

**Pull request closed without merging**